templates/index.html

```diff
@@ -26,7 +26,7 @@ <h1>A simple extensible CSS framework</h1>
     <div class="col-4">
       <div class="p-heading-icon">
         <div class="p-heading-icon__header is-stacked">
-          <img class="b-lazy p-heading-icon__img" src=data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw== data-src="https://assets.ubuntu.com/v1/1958451f-pictogram_lightweight01b-dark.svg" alt=""/>
+          <img class="b-lazy p-heading-icon__img" src=data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==" data-src="https://assets.ubuntu.com/v1/1958451f-pictogram_lightweight01b-dark.svg" alt=""/>
```

Comment:
You added closing quote, but not the starting one:

```suggestion
          <img class="b-lazy p-heading-icon__img" src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==" data-src="https://assets.ubuntu.com/v1/1958451f-pictogram_lightweight01b-dark.svg" alt=""/>
```

BTW, in the "Files changed" tab on GitHub you should be able to batch all the suggestions to one commit instead of comitting them separately:
<img width="722" alt="Screenshot 2020-09-22 at 10 40 54" src="https://user-images.githubusercontent.com/83575/93861169-62385200-fcc0-11ea-93c3-86ea5baa77fe.png">